README.md

```diff
@@ -14,6 +14,21 @@ The project was *initially* forked from the  upstream [mozilla-services/go-cose]
 
 We thank the [Mozilla maintainers and contributors][mozilla-contributors] for their great work that formed the base of the [veraison/go-cose][veraison-go-cose] project.
 
+## Community
+
+The [veraison/go-cose](https://github.com/veraison/go-cose) project is an open source community effort.
+
+You can reach the go-cose community via::
+
+- [Mailing List](https://groups.io/g/go-cose-community)
+- Bi-weekly meetings: 08:00-09:00 Pacific
+  - [Zoom meeting link](https://us02web.zoom.us/j/81054434992?pwd=YjNBU21seU5VcGdtVXY3VHVjS251Zz09)
+  - [Calendar ics link](https://zoom.us/meeting/tZUtcu2srT8jE9YFubXn-lC9upuwUiiev52G/ics)
+- [Meeting Notes](https://hackmd.io/9dpXsrBxTrefxrNaTgq12Q)
+- [Meeting Recordings](https://www.youtube.com/@go-cose-community3000)
```

Comment:
We already record meeting minutes under veraison zulip chat every week, so would continue to do so.